pkg/knuu/knuu.go

```diff
@@ -223,3 +221,8 @@ func setupProxy(ctx context.Context, k *Knuu) error {
 	k.Logger.Debugf("Proxy endpoint: %s", endpoint)
 	return nil
 }
+
+func setupMinio(ctx context.Context, k *Knuu) (err error) {
+	k.MinioClient, err = minio.New(ctx, k.K8sClient)
+	return err
+}
```

Comment:
Building on comment, we wouldn't need this. This would become part of test helpers or something.